akka-typed-tests/src/test/java/akka/typed/javadsl/ActorCompile.java

```diff
@@ -56,6 +59,13 @@ public MyMsgB(String greeting) {
     });
   }
 
+  {
+    Behavior<MyMsg> b = Actor.withTimer(timers -> {
```

Comment:
naming nitpick: perhaps this is `withTimers` as it does not start a timer, but give you the capability to do so?